src/tests/FunctionalTests/Android/Device_Emulator/JIT/Program.cs

```diff
@@ -8,6 +8,7 @@ public class Program
     public static int Main(string[] args)
     {
         Console.WriteLine("Hello, Android!"); // logcat
-        return 42;
+        int result = ((string)AppContext.GetData("test_runtimeconfig_json")).Equals("25") ? 42 : 1;
```

Comment:
I think this would better be separate test

Reply:
Will split the test in a followed-up PR.

Reply:
Agreed re: separating it